### PR TITLE
Test verify email thera

### PR DESCRIPTION
**Pull request by TheraFu**
Looks like something's wrong and now it includes what I pulled from master...

### CODE REVIEW

File: pages/accounts/profile.py [hunk 1]
```diff
@@ -193,6 +193,10 @@ class Emails(Region):
 
         _root_locator = (By.XPATH, '//div[div[contains(text(),"Emails")]]')
         _email_locator = (By.CSS_SELECTOR, '.info > .email-entry')
+        _unverified_locator = (By.CLASS_NAME, 'unconfirmed-warning')
+        _add_email_locator = (By.ID, 'add-an-email')
+        _email_form_locator = (By.CSS_SELECTOR, '.editable-input input')
+        _email_submit_locator = (By.XPATH, '//button[@type="submit"]')
```
Comment (gregfitch) [inline pages/accounts/profile.py]: Instead of the XPath, try a selector here:
(By.CSS_SELECTOR, '[type=submit]')

File: pages/utils/email.py [hunk 2]
```diff
@@ -139,7 +144,12 @@ def emails(self):
     def compose(self):
         """Return a composition page."""
         self.find_element(*self._compose_locator).click()
-        return Compose(self.selenium)
+        return Compose(self.driver)
+
+    @property
+    def openedmail(self):
```
Comment (gregfitch) [inline pages/utils/email.py]: python names should be words separated with an underscore:
openedmail => opened_mail

File: pages/utils/email.py [hunk 3]
```diff
@@ -139,7 +144,12 @@ def emails(self):
...
+        """Return a opened email region."""
+        return self.Openedmail(self)
```
Comment (gregfitch) [inline pages/utils/email.py]: Class names should be Proper case:
Openedmail => OpenedMail

File: pages/utils/email.py [hunk 4]
```diff
@@ -211,6 +221,7 @@ class Emails(Region):
 
         _subject_locator = (By.CLASS_NAME, 'td3')
         _excerpt_locator = (By.CLASS_NAME, 'email-excerpt')
+        _emails_locator = (By.XPATH, '//tbody/tr')
```
Comment (gregfitch) [inline pages/utils/email.py]: Switch this to a selector: 'tbody > tr'

File: tests/conftest.py [hunk 5]
```diff
@@ -10,6 +10,8 @@
 load_dotenv(dotenv_path=DOTENV_PATH)
 
 # Import fixtures from our package so pytest can detect them
+
+
```
Comment (gregfitch) [inline tests/conftest.py]: remove these blank lines
shouldn't have blanks between the comment ("# Import...") and the affected code ("pytest_...")

File: tests/accounts/test_accounts_profile.py [hunk 6]
```diff
@@ -122,15 +123,57 @@ def test_profile_email_fields(accounts_base_url, selenium):
 @accounts
 def test_verify_an_existing_unverified_email(accounts_base_url, selenium):
     """Test the user email verification process."""
-
-
+    page = GuerrillaMail(selenium).open()
```
Comment (m1yag1) [inline tests/accounts/test_accounts_profile.py]: Tests need to use the Given, When, Then structure as stated in issue #12

**Pull request merged**